src/azure-cli/azure/cli/command_modules/vm/_help.py

```diff
@@ -224,7 +225,7 @@
         # create and update template object in local cli cache. Defers put request to ARM
         # Cache object ttl set via az configure.
         az image builder create --image-source $imagesource -n mytemplate \\
-            -g my-group --scripts $script --defer
+            -g my-group --scripts $script --identity myidentity --defer
```

Comment:
I'd recommend adding **identity creation command** before this command.

Reply:
Good advice